unmarshal_test.go

```diff
@@ -24,15 +24,23 @@ func ExampleUnmarshal() {
 	TestBinary := []byte{
 		0x1a, 0x45, 0xdf, 0xa3, // EBML
 		0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x10, // 0x10
-		0x42, 0x82, 0x85, 0x77, 0x65, 0x62, 0x6d, 0x00,
-		0x42, 0x87, 0x81, 0x02, 0x42, 0x85, 0x81, 0x02,
+		0x42, 0x82, 0x85, 0x77, 0x65, 0x62, 0x6d, 0x00, // EBMLDocType
+		0x42, 0x87, 0x81, 0x02, // EBMLDocTypeVersion
+		0x42, 0x85, 0x81, 0x02, // EBMLDocTypeReadVersion
+		0x18, 0x53, 0x80, 0x67, 0x85, // Segment
+		0x2a, 0xd7, 0xb1, 0x81, 0x03, // TimecodeScale
 	}
 	type TestEBML struct {
 		Header struct {
+			Metadata
```

Comment:
Marshalling embedded struct outputs flattened result in standard encoding/json. (https://play.golang.org/p/orp7cvQplU)
It would be better to be non-embedded field.

I'd like to add a validation of given struct in future. (#23)
Adding a tag `ebml:"-"` would make it easy to validate.

Reply:
Make sense, I've changed all metadata to be unembedded field and add `ebml:"-"`
in eb899b7872964e34ae4681fc8af4bff25fbce856